reader-gtfs/src/main/java/com/graphhopper/gtfs/WrapperGraph.java

```diff
@@ -112,6 +112,12 @@ public EdgeIteratorState getEdgeIteratorState(int edgeId, int adjNode) {
         }
     }
 
+    @Override
+    public EdgeIteratorState getEdgeIteratorStateForKey(int edgeKey) {
+        // todonow?
```

Comment:
@michaz, can you check `WrapperGraph` here? I was not sure what is and what is not needed here.

Reply:
postponed until we need this method